docs/source/technical_tutorials/working_with_r/access_cmr_r.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<p>Should we reference the page about S3 access to help people understand the when?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/482/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Is it not possible to read netcdf (or hdf) files directly from S3 in R?</p>

Reply:
MODIS is HDF4, it's not possible with this.